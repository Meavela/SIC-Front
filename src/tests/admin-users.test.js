import React from 'react';
import renderer from 'react-test-renderer';
import AdminUsers from '../components/admin-users';
import { BrowserRouter } from 'react-router-dom';

it('admin user page correctly', () => {
    const tree = renderer
      .create(
        <BrowserRouter>
            <AdminUsers />
        </BrowserRouter>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });