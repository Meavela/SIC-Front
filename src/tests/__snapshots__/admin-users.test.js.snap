// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`admin user page correctly 1`] = `
Array [
  <a
    href="/admin"
    onClick={[Function]}
  >
    â¬… Back
  </a>,
  <div
    className="contentForm"
  >
    <h1
      className="title"
    >
      Admin Users Page - Users
    </h1>
    <div
      className="createContent"
    >
      <form
        className=""
      >
        <div
          className="form-group"
        >
          <label
            className="form-label"
            htmlFor="formBasicUsername"
          >
            Username :
          </label>
          <input
            className="form-control"
            id="formBasicUsername"
            onChange={[Function]}
            placeholder="Enter username"
            type="text"
          />
          <small
            className="text-muted form-text"
          />
        </div>
        <div
          className="form-group"
        >
          <label
            className="form-label"
            htmlFor="formBasicPassword"
          >
            Password :
          </label>
          <input
            className="form-control"
            id="formBasicPassword"
            onChange={[Function]}
            placeholder="Enter Password"
            type="password"
          />
        </div>
        <div
          className="errorContent form-group"
        >
          <label
            className="form-label"
          >
            
          </label>
        </div>
        <div
          className="submitButton form-group"
        >
          <button
            className="btn btn-primary"
            disabled={false}
            onClick={[Function]}
            type="button"
          >
            Create
          </button>
        </div>
        <div
          className="successContent form-group"
        >
          <label
            className="form-label"
          >
            
          </label>
        </div>
      </form>
    </div>
  </div>,
  <div
    className="contentTable"
  >
    <table
      className="table table-blue table-striped table-bordered table-hover"
    >
      <thead>
        <tr>
          <th>
            Id
          </th>
          <th>
            Username
          </th>
        </tr>
      </thead>
      <tbody />
    </table>
  </div>,
]
`;
